ab_proxying.py

```diff
@@ -68,7 +68,10 @@ def redirect_ab (request):
     user_identifier = current_user(request) ['username'] or str (session['session_id'])
 
     # This will send either % PROXY_TO_TEST_PROPORTION of the requests into redirect, or 50% if that variable is not specified.
-    redirect_proportion = int (os.getenv ('PROXY_TO_TEST_PROPORTION', '50'))
```

Comment:
I don't see how setting `PROXY_TO_TEST_PROPORTION=0` would cause this whole expression to return 50?

Testing shows it doesn't:

```
$ env PROXY_TO_TEST_PROPORTION=0 python3 -c 'import os; print(int(os.getenv("PROXY_TO_TEST_PROPORTION", "50")))'
0
```

Reply:
You're right. I had tested it but I think I applied the `int` operation in the wrong way. I just re-tested and you're right. Closing the PR now, thank you for spotting this!

Reply:
I think you might have mistyped the variable name?